### PR TITLE
[WIP] Allow displaying offline users in MUC roster

### PR DESCRIPTION
**Pull request by benlangfeld**
If we've already populated the roster from the room members list, then the user coming online will look like a role/affiliation change.

### CODE REVIEW

**Comment by benlangfeld** [comment]
@melissanoelle / @bklang: Review?

File: src/view/pane/roster.js [hunk 1]
```diff
@@ -69,52 +69,15 @@ Candy.View.Pane = (function(self, $) {
...
-              }
-              return true;
-            });
-          }
-          // first user in roster
-          if(!userInserted) {
-            rosterPane.append(html);
-          }
-
+          self.Roster._insertUser(roomJid, roomId, user, userId, currentUser);
```
Comment (bklang) [inline src/view/pane/roster.js]: *SO* Much better.

**Comment by bklang** [comment]
I'll run this out to DevM for testing.

**Comment by lkraav** [comment]
I have trouble understanding what does this mean for the final user experience? People's names will be always listed with a status indicator or...?

**Comment by benlangfeld** [comment]
No, this makes no difference for Candy by default. It means that if someone manipulates the roster in a plugin, as we do, then those roster items are sorted correctly and do not break the built-in functionality.

**Pull request merged**